packages/core/src/examples/healthchecks.ts

```diff
@@ -0,0 +1,56 @@
+import express from 'express';
+import { startServer } from '../lib/server';
+import { defaultHealthCheck } from '../config/defaults';
+
+const app = express();
+
+console.log('Server with a basic enabled health check');
+startServer(app, {
+  port: 4999,
+  healthCheck: {
+    enabled: true,
+  },
+});
+// {"status":"ok"}
+
+console.log('Server with a disabled health check');
```

Comment:
This would be the same as not passing it along, right?

Reply:
That's correct, I prefered to keep them both